docs/network-requirements.md

```diff
@@ -0,0 +1,16 @@
+# Network Requirements
+
+Antrea has a few network requirements to get started, ensure that your hosts and firewalls allow the necessary traffic based on your configuration.
+
+| Configuration              | Host(s)                          | ports/protocols   |
+| -------------------------- | -------------------------------- | ----------------- |
+| Antrea with VXLAN enabled  | All                              | UDP 4789          |
+| Antrea with Geneve enabled | All                              | UDP 6081          |
+| Antrea with STT enabled    | All                              | TCP/UDP 1607      |
+| Antrea with GRE enabled    | All                              | UDP 4754          |
```

Comment:
I remember GRE in Antrea is not MAC over UDP. Could you check this from OVS doc or capturing tunnel traffic on a live cluster? I can confirm VXLAN and Geneve ports are correct, but not sure others.

Reply:
It is definitely not UDP. We just need to allow GRE traffic.

Reply:
GRE is a standalone IP protocol, id=47.